src/main/java/com/google/maps/model/PlaceDetails.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps.model;
+
+import java.net.URL;
+
+/**
+ * PlaceDetails is the result of a Place Details request. A Place Details request returns more
+ * comprehensive information about the indicated place such as its complete address, phone number,
+ * user rating and reviews.
+ *
+ * See <a href="https://developers.google.com/places/web-service/details#PlaceDetailsResults">
+ *   Place Details Results</a> for more detail.
+ */
+public class PlaceDetails {
+
+  /** addressComponents is a list of separate address components used to compose a given address. */
+  public AddressComponent[] addressComponents;
+
+  /** formattedAddress is a string containing the human-readable address of this place. */
+  public String formattedAddress;
+
+  /** formattedPhoneNumber contains the place's phone number in its local format. */
+  public String formattedPhoneNumber;
+
+  /** adrAddress is an undocumented address field.
+   *
+   * @deprecated Undocumented field.
+   */
+  @Deprecated
+  public String adrAddress;
+
+  /** geometry is the location of the Place. */
+  public Geometry geometry;
+
+  /** icon contains the URL of a suggested icon which may be displayed to the user when indicating
```

Comment:
nit: there are a few comments around that are multi-line and start on the /** line, can you move them so the /** has no following text, unless it all fits on one line?